internal/app/exporter/exporter.go

```diff
@@ -235,6 +236,10 @@ func NewCollector(logger slog.Logger, collectorConfiguration Configuration, awsA
 			"Number of active sessions where the wait event type is not CPU",
 			[]string{"aws_account_id", "aws_region", "dbidentifier"}, nil,
 		),
+		transactionLogsDiskUsage: prometheus.NewDesc("rds_transaction_log_disk_usage_bytes",
```

Comment:
Refer to `logs` vs `log` comment:

```suggestion
		transactionLogsDiskUsage: prometheus.NewDesc("rds_transaction_logs_disk_usage_bytes",
```

Reply:
Correct! I updated it